### PR TITLE
Add  unit testing

### PR DESCRIPTION
**Pull request by victorvarza**

### CODE REVIEW

File: api/tests/test_api.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    ]
+
+    for metric in expected_metrics:
+        metric in metrics
+
+    teardown_redis(redis_container)
+
+
+def setup_redis():
+    redis_container = RedisContainer(image="ghcr.io/hlesey/redis:4").with_bind_ports(6379,6379)
```
Comment (adriananeci) [inline api/tests/test_api.py:64]: Nice 😍

**Review by adriananeci — APPROVED** [review]
Small nits, otherwise LGTM

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,4 @@
 .idea
-.vscode
```
[the remark this answers is not in the document]
Comment (victorvarza) [reply .gitignore:2]: I wanted to have somewhere saved the IDE configuration of how to run this project locally.

[the remark this answers is not in the document]
Comment (adriananeci) [reply .gitignore:2]: Not everyone is using vscode. This config might be moved on a different repo where we can aggregate config for multiple IDEs if these configs bring any value

**Pull request merged**